### ISSUE TITLE
BLACS plugin initialization not Python 2 compatible

### ISSUE THREAD

**Issue by philipstarkey** [labscript-suite-oldfinal1/blacs#35]
**Original report ([archived issue](https://labscript-suite.github.io/labscript-suite-bitbucket-archive/#!/labscript_suite/blacs/issues/35)) by David Meyer (Bitbucket: [dihm](https://bitbucket.org/%7B8df655d6-8661-4957-b4ba-669cef19bf2d%7D/), GitHub: [dihm](https://github.com/dihm)).**

----------------------------------------
I'm getting an error starting BLACS after merging in up to 828 with my old Py2/Qt4 setup:

```
#!python

Traceback (most recent call last):
  File "C:\labscript_suite\blacs\__main__.py", line 148, in <module>
    import plugins
  File "C:\labscript_suite\blacs\plugins\__init__.py", line 34, in <module>
    if not module_name in exp_config['BLACS/plugins']:
AttributeError: LabConfig instance has no attribute '__getitem__'
```

It looks like some Python3 configparser syntax worked its way in on line 34 of plugins/\__init__.py

**Comment by philipstarkey** [comment]
**Original comment by Chris Billington (Bitbucket: [cbillington](https://bitbucket.org/%7Be363c5a9-5075-4656-afb5-88bd6a6dceeb%7D/), GitHub: [chrisjbillington](https://github.com/chrisjbillington)).**

----------------------------------------
Ah, we are all on Python 2 since BLACS hasn't been ported yet, but this is caused by the fact that the configparser module from Python 3 has been backported to Python 2 and is available as a separate package, which clearly some of us have installed.

labconfig does the following in order to work in both Python 2 and Python 3:


```
#!python

try:
    import configparser
except ImportError:
    # Python 2
    import ConfigParser as configparser
```

But the `configparser` module exists for those of us in Python 2 who have the backported module.

Rather than require the backported module I'll just fix the syntax to be `ConfigParser` compatible.

**Comment by philipstarkey** [comment]
**Original comment by Chris Billington (Bitbucket: [cbillington](https://bitbucket.org/%7Be363c5a9-5075-4656-afb5-88bd6a6dceeb%7D/), GitHub: [chrisjbillington](https://github.com/chrisjbillington)).**

----------------------------------------

* changed state from "new" to "resolved"

Fixed by [pull request #27](https://bitbucket-archive.labscriptsuite.org/#!/labscript_suite/blacs/pull-requests/27)
